app/src/main/java/com/androidavanzado/covid19stadistics/dependency/module/RepositoryModule.kt

```diff
@@ -0,0 +1,27 @@
+package com.androidavanzado.covid19stadistics.dependency.module
+
+import com.androidavanzado.covid19stadistics.model.Repository
+import com.androidavanzado.covid19stadistics.storage.remote.RemoteDataSource
+import com.androidavanzado.covid19stadistics.storage.remote.ServicesApiInterface
+import dagger.Module
+import dagger.Provides
+import javax.inject.Singleton
+
+
+/*object Injector {
```

Comment:
if this is not going to be used in the future better remove it 👍

Reply:
done